### PR TITLE
Title only shown after question answered and feedback shown updates (re #682)

### PR DESCRIPTION
**Pull request by Sauder91**

### CODE REVIEW

File: js/adapt-question-feedback.js [hunk 1]
```diff
@@ -79,6 +79,7 @@ define(["coreJS/adapt", "coreViews/componentView"], function(Adapt, ComponentVie
             }
 
             this.model.set("feedback", questionModel.get("feedbackMessage"));
+            this.render();
```
Comment (mediabounds) [inline js/adapt-question-feedback.js:82]: @Sauder91 I don't think it is necessary to call render here--at line 8, a listener is set up to automatically re-render the component when feedback changes.

File: templates/question-feedback.hbs [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <div class="question-feedback-inner component-inner" role="region" aria-label="{{_globals._components._text.ariaRegion}}">
 	<div class="{{_component}}-header component-header">
+		{{#if feedback}}
```
Comment (mediabounds) [inline templates/question-feedback.hbs:3]: @Sauder91 Can you fix the indentation here and at line 42?

**Review by mediabounds — CHANGES_REQUESTED** [review]

File: js/adapt-question-feedback.js (hunk 1 above)
Comment (Sauder91) [reply js/adapt-question-feedback.js:82]: I originally had it without adding this render call and it wasn't updating.  Maybe the change:feedback isn't being triggered except on the first update?

Comment (mediabounds) [reply js/adapt-question-feedback.js:82]: Okay, let's keep this and then remove the change listener.

**Pull request closed without merging**